### DISCUSSION TITLE
Add Slidev as a Backend

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#5415] [Feature Requests]
It would be really great to have [Slidev](https://sli.dev/) as a slide backend in addition to RevealJS.

**Comment by cderv** [comment]
Revealjs was easy to add for us because it is among Pandoc supported format. https://pandoc.org/index.html
Slidev is not. It would be more work, but could be interesting to add yes as a new slide format! Thanks for the idea!

**Reply by RoyiAvital** [reply]
Great, made me optimistic :-).

**Reply by cderv** [reply]
It would be interesting for us to understand also what is missing in the Quarto HTML presentation format built with revealjs. 

It is even possible to already use the capabilities of Quarto and Pandoc to manage to take a `.qmd` and output the right `.md` for you to use sli.dev. You can also try to leverage [extensions](https://quarto.org/docs/extensions/) to built a custom format to package it. 
Worth a try

**Reply by RoyiAvital** [reply]
Interesting indeed.

What I miss in RevealJS, mostly, is the easiness of creating a complex layout and object animations.  
For instance, this: https://github.com/quarto-dev/quarto-cli/discussions/3559.
Though, I think, this one might be solved at Quarto level.

**Reply by RoyiAvital** [reply]
At the end, I want to be free to locate elements anywhere I want and just set their size and Z index. Similar to what you can do in PowerPoint and such.

It seems to have more of this ability then RevealJS on Quatro.

**Comment by BinaryQuantumSoul** [comment]
Any news on this?
This or the marp one https://github.com/quarto-dev/quarto-cli/discussions/7377 could be done with simple replace rules right? Maybe with extensions?

**Reply by mcanouil** [reply]
This is not a short/mid-term priority.

Note that if you think a "simple rule" (i don't know what that means in this context) is what it takes, you could make a pull request.

**Reply by RoyiAvital** [reply]
@BinaryQuantumSoul , I opened a feature request on `slidev` as well: https://github.com/slidevjs/slidev/issues/1986. Maybe some support there will increase the probability of the `slidev` community driving some support.

**Reply by BinaryQuantumSoul** [reply]
> This is not a short/mid-term priority.
> 
> Note that if you think a "simple rule" (i don't know what that means in this context) is what it takes, you could make a pull request.

I will look into PRing then

**Reply by BinaryQuantumSoul** [reply]
I also opened an issue on pandoc since if it does support it, quarto and others will automatically support it as well
https://github.com/jgm/pandoc/issues/10483

**Comment by mcanouil** [comment]
FYI

- https://github.com/jgm/pandoc/issues/10483
